bin/build_offload.sh

```diff
@@ -131,7 +134,11 @@ if [ "$1" != "nocmake" ] && [ "$1" != "install" ] ; then
       fi
    fi
       if [ "$AOMP_BUILD_SANITIZER" == 1 ]; then
-        ASAN_CMAKE_OPTS="$COMMON_CMAKE_OPTS -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/asan/cmake -DSANITIZER_AMDGPU=1 -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PER_TARGET_RUNTIME_DIR=OFF $AOMP_ASAN_ORIGIN_RPATH"
+        if [ "$AOMP_STANDALONE_BUILD" == 1 ]; then
+          ASAN_CMAKE_OPTS="$COMMON_CMAKE_OPTS -DCMAKE_PREFIX_PATH=$ROCM_CMAKECONFIG_PATH;$AOMP_INSTALL_DIR/lib/asan/cmake -DSANITIZER_AMDGPU=1 -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PER_TARGET_RUNTIME_DIR=OFF $AOMP_ASAN_ORIGIN_RPATH"
```

Comment:
Same for ASAN_CMAKE_OPTS also we can apply comment https://github.com/ROCm/aomp/pull/917/files#r1593473768.

One less concern here ```ROCM_CMAKECONFIG_PATH``` is set up by build_openmp_extras.sh script explicitly. So what I am seeing here for non-standalone builds duplicate values in ```CMAKE_PREFIX_PATH```

Reply:
I think we can get rid of `$AOMP_INSTALL_DIR/lib/asan/cmake` where `ROCM_CMAKECONFIG_PATH` is used for asan builds as it does duplicate the path for `AOMP_STANDALONE=0` builds.